api/views.py

```diff
@@ -436,6 +537,14 @@ def root_settings(request):
 
 @api_view(['POST'])
 def upload_sample_input(request):
+    """
+    Creates a sample input. Only image input is supported currently.
+
+    The data passed into the POST request needs to contain:
+        demo_id (str): The demo that the sample input should belong to.
+        sample-image* (file): A value with its key having 'sample-image'
```

Comment:
sample-image-*

Reply:
Hmm but sample-image is the prefix required, not sample-image-

Reply:
Actually I'm not at all sure how it should be written, so just wrote some regex like thing

Reply:
But that makes more sense actually